source/Generic/GamesSizeCalculator/Steam/SteamSizeCalculator.cs

```diff
@@ -9,84 +10,160 @@
 
 namespace GamesSizeCalculator.SteamSizeCalculation
 {
-    public class SteamSizeCalculator
+    public class SteamSizeCalculator : IOnlineSizeCalculator
     {
         private ILogger logger = LogManager.GetLogger();
         public ISteamApiClient SteamApiClient { get; }
+        public ISteamAppIdUtility SteamAppIdUtility { get; }
+        public bool IncludeDLC { get; set; }
+        public bool IncludeOptional { get; set; }
+        public bool HandleNonSteamGames { get; set; }
+        public string[] RegionalWords { get; set; }
+        public string[] RegionalWordsBlacklist { get; set; }
+        public string ServiceName { get; } = "Steam";
```

Comment:
Should be a constant. Also properties that are not change later should be set to `readonly`

Reply:
It's an interface member, so it can't be static, and properties can't be readonly in C# 7.3

Reply:
That's true, I didn't notice